### ISSUE TITLE
Allow tag for release to be passed as an argument

### ISSUE THREAD

**Issue by leo** [zeit/release#85]

**Comment by bfred-it** [comment]
Does this mean that I would be able to add releases to previous tags?

**Comment by leo** [comment]
Yep, correct! This should still be possbile, as you can reset the git HEAD to that old release, but we'll make it even easier.
